src/app/loginComponents/accounts/controls/controls.component.html

```diff
@@ -1,8 +1,10 @@
 
 <div class="p-3">
-  <mat-card class="alert alert-danger" role="alert">
-    <mat-icon>error</mat-icon> Dockstore account controls have a large impact and should be used with extreme caution.
-  </mat-card>
+  <div *ngIf="(canChangeUsername$ | async)">
```

Comment:
You can just put the `*ngIf` directly on the `mat-card` and get rid of this `<div>`.